### ISSUE TITLE
batch scanImage only returns value of first qr-code

### ISSUE THREAD

**Issue by Pohalo** [nimiq/qr-scanner#149]
Hi, 
i try to scan some qr-code-images iterating the image array.
The first scanImage is working and returns the right link.
But all following scans always return link of the first qr-code.

What is my problem?

```
var image = ["img1.jpg","img2.jpg","img3.jpg"];
$.each( image, function( key, value ) {
     		fileQrResult = document.getElementById('result');
     		var file = "qr/"+value;
	QrScanner.scanImage(file, { returnDetailedScanResult: true })
                .then(result => setResult(fileQrResult, result, key, value))
                .catch(e => setResultError(fileQrResult, e || 'No QR code found.'));
        });
```

**Comment by danimoh** [comment]
Thanks for this report.
This was actually a bug.
It's fixed now in version 1.4.1.

A little tip:
If you're planning to scan a lot of images, passing a `qrEngine` and `canvas` to be reused between the `scanImage` calls, avoids the overhead of recreating them for each individual call, as described in the [readme](https://github.com/nimiq/qr-scanner#single-image-scanning).
However, if you want to run multiple scans in parallel, on multiple threads, either don't pass a `qrEngine` as you're doing it right now or create a pool of multiple workers which are reused in turn.

**Comment by Pohalo** [comment]
Thank you for fixing it and for your tip!
